server/ControlPlane/Compute/Kubernetes/RunFinalizer.cs

```diff
@@ -199,7 +199,15 @@ private async Task FinalizeRun(ObservedRunState runState, CancellationToken canc
         }
 
         await _repository.UpdateRunAsFinal(runState.Id, cancellationToken);
```

Comment:
We should make UpdateRunAsFinal return the necessary information.

Reply:
Updated. @johnstairs it would be good if you could check my handywork here. I am pulling the time stamps out as part of the update query, however, the startedAt and finishedAt timestamps are not in the main table but in the jsonb run field and I am not sure if this extraction is actually more efficient than just doing a GetRun after it is done.